### PR TITLE
Additional content and routing

### PR DESCRIPTION
**Pull request by iwootten**
Update content and routing as follows:

- HH - E/W - type-of-flat - add new answer option after 'Part of a converted or shared house'. Label to be 'Part of another converted building' and answer guidance to be 'For example, former school, church or warehouse' . No routing or validation changes
- HH - E/W - number-of-vehicles - change answer option '4 or more' to be '5 or more'. Add new answer option above with label '4'. No changes to routing or validation
- HH/HI - E/W - term-time-address-country - if unanswered then routing should be to end of section
- HH/HI - E/W - apprenticeship - include this question in the routing to other-qualifications. Routing should be to other-qualifications if answers to apprenticeship, degree, nvq-level, a-level and gcse are no or not answered
- HH/HI - E/W - armed-forces. Change to mutually exclusive answer type. 

### Checklist 

Check schemas have been updated as per above.

### CODE REVIEW

**Comment by iwootten** [comment]
[[A] Additional Content and routing to current questions (S)](https://trello.com/c/BGwL5wUJ/3605-a-additional-content-and-routing-to-current-questions-s)

**Comment by petechd** [comment]
Shouldn't this twisty content change too?
https://github.com/ONSdigital/eq-questionnaire-schemas/blob/634138d859a0c70f4aabc919d292472f6215df0c/source/jsonnet/england-wales/individual/blocks/employment/armed_forces.jsonnet#L26

**Comment by iwootten** [comment]
> Shouldn't this twisty content change too?
> https://github.com/ONSdigital/eq-questionnaire-schemas/blob/634138d859a0c70f4aabc919d292472f6215df0c/source/jsonnet/england-wales/individual/blocks/employment/armed_forces.jsonnet#L26

Possibly, though I need glasses to be able to read the content of the screenshot?

**Comment by MebinAbraham** [comment]
A quick-launch link would be handy.

**Comment by iwootten** [comment]
> A quick-launch link would be handy.

Sure, it would. You're welcome to provide one 😉

File: source/jsonnet/england-wales/individual/blocks/employment/armed_forces.jsonnet [hunk 1]
```diff
@@ -11,25 +11,26 @@ local question(title) = {
...
       id: 'armed-forces-answer',
       mandatory: false,
+      type: 'Checkbox',
       guidance: {
         show_guidance: 'Why your answer is important',
         hide_guidance: 'Why your answer is important',
         contents: [
           {
-            description: 'We are measuring the number of people who have served in the UK Armed Forces and have now left. Government and councils need this information to carry out their commitments made under the Armed Forces Covenant. This is a promise by the nation ensuring that those who serve or who have served in the armed forces, and their families, are not disadvantaged.',
+            description: 'Your answer will help your local community by providing information needed to support people who used to serve in the UK Armed Forces but have left.',
```
Comment (petechd) [inline source/jsonnet/england-wales/individual/blocks/employment/armed_forces.jsonnet]: Not exactly as it should be: "people who used to serve in the UK Armed Forces but have now left" & "This is a promise by the nation to ensure..." below

File: source/jsonnet/england-wales/individual/blocks/employment/armed_forces.jsonnet [hunk 2]
```diff
@@ -11,25 +11,26 @@ local question(title) = {
...
+          },
+          {
+            description: 'Councils and government will use this information to carry out the commitments they made under the Armed Forces Covenant. This is a promise by the nation ensuring that those who serve or who have served in the UK Armed Forces, and their families, are not disadvantaged.',
```
Comment (petechd) [inline source/jsonnet/england-wales/individual/blocks/employment/armed_forces.jsonnet]: "This is a promise by the nation **to ensure**"

**Comment by iwootten** [comment]
So the content changes aren't applicable to this card - they are all subject to review and covered here: https://trello.com/c/5NWYjTTQ/3517-a-content-and-validation-changes

**Comment by iwootten** [comment]
Going to revert all the content changes made.

**Pull request merged**